packages/paste-website/src/pages/components/skeleton-loader/index.mdx

```diff
@@ -0,0 +1,227 @@
+---
+title: Skeleton Loader - Components
+description: A component that gives users a hint of what type of information will be loaded on a page and helps reduce the perceived time a page takes to load.
+slug: /components/skeleton-loader/
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Avatar} from '@twilio-paste/avatar';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {Card} from '@twilio-paste/card';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {SkeletonLoader} from '@twilio-paste/skeleton-loader';
+import {Stack} from '@twilio-paste/stack';
+import {Table, THead, TBody, Tr, Td, Th} from '@twilio-paste/table';
+import {Text} from '@twilio-paste/text';
+import {CalendarIcon} from '@twilio-paste/icons/esm/CalendarIcon';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {
+  CompositionExample,
+  TextExample,
+  ParagraphExample,
+  HeadingExample,
+  ButtonsExample,
+  AvatarExample,
+  IconExample,
+  CardExample,
+  TableExample,
+} from '../../../component-examples/SkeletonLoaderExamples';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/skeleton-loader"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/skeleton-loader/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Skeleton Loader"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Skeleton Loader"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/skeleton-loader"
+  storybookUrl="/?path=/story/components-skeleton-loader--default"
+  data={props.data.allPasteComponent.edges}
+  packageStatus={props.data.allAirtable.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
```

Comment:
Similarly to Alerts, we're going to start adding a basic default example to the top of every component page so you see the component up front.

You could probably just get away with reusing the default example.